plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/jdbc/TestIcebergJdbcCatalogConnectorSmokeTest.java

```diff
@@ -60,6 +73,14 @@ protected QueryRunner createQueryRunner()
         warehouseLocation = Files.createTempDirectory("test_iceberg_jdbc_catalog_smoke_test").toFile();
         closeAfterClass(() -> deleteRecursively(warehouseLocation.toPath(), ALLOW_INSECURE));
         TestingIcebergJdbcServer server = closeAfterClass(new TestingIcebergJdbcServer());
+        jdbcCatalog = (JdbcCatalog) buildIcebergCatalog("tpch", ImmutableMap.<String, String>builder()
+                        .put(CATALOG_IMPL, JdbcCatalog.class.getName())
```

Comment:
nit: I'd rather avoid using static imports for these properties. Their names are rather generic and this can turn out to be  error prone in the maintenance.

Reply:
`CATALOG_IMPL` comes from `CatalogProperties` class. I don't feel qualified access improves anything.